### PR TITLE
Tileset content.url to content.uri

### PR DESCRIPTION
**Pull request by ggetz**
Part of #6697

To reflect updates to the [3D Tiles spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/301), deprecated support for 3D Tiles `content.url` in favor of `content.uri`.

Also added a test to make sure tilesets using data uri's for their content load and render.

TODO:
 - [ ] Once the PR is merged, add a note to #5363.
- [ ] Update sample tilesets

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Change Log
 * Dropped support for directory URLs when loading tilesets to match the updated [3D Tiles spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/272). [#6502](https://github.com/AnalyticalGraphicsInc/cesium/issues/6502)
 * KML and GeoJSON now use `PolylineGraphics` instead of `CorridorGraphics` for polylines on terrain. [#6706](https://github.com/AnalyticalGraphicsInc/cesium/pull/6706)
 
+* Support for 3D Tiles `content.url` is deprecated to reflect updates to the [3D Tiles spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/301) and will be removed in Cesium 1.50. Use `content.uri instead`. [#6744](https://github.com/AnalyticalGraphicsInc/cesium/pull/6744)
```
Comment (hpinkos) [inline CHANGES.md]: whitespace above this

File: CHANGES.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ Change Log
 * Dropped support for directory URLs when loading tilesets to match the updated [3D Tiles spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/272). [#6502](https://github.com/AnalyticalGraphicsInc/cesium/issues/6502)
 * KML and GeoJSON now use `PolylineGraphics` instead of `CorridorGraphics` for polylines on terrain. [#6706](https://github.com/AnalyticalGraphicsInc/cesium/pull/6706)
 
+##### Deprecated :hourglass_flowing_sand:
+* Support for 3D Tiles `content.url` tileset JSON property is deprecated to reflect updates to the [3D Tiles spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/301) and will be removed in Cesium 1.50. Use `content.uri instead`. [#6744](https://github.com/AnalyticalGraphicsInc/cesium/pull/6744)
+
```
Comment (lilleyse) [inline CHANGES.md]: I was late for 1.47, so just move up to the 1.48 section.

File: Specs/Scene/Cesium3DTilesetSpec.js [hunk 3]
```diff
@@ -117,6 +117,8 @@ defineSuite([
     var tilesetReplacementWithViewerRequestVolumeUrl = 'Data/Cesium3DTiles/Tilesets/TilesetReplacementWithViewerRequestVolume/tileset.json';
 
     var tilesetWithExternalResourcesUrl = 'Data/Cesium3DTiles/Tilesets/TilesetWithExternalResources/tileset.json';
+    var tilesetUrlWithContentUri = 'Data/Cesium3DTiles/Tilesets/TilesetWithContentUri/tileset.json';
```
Comment (lilleyse) [inline Specs/Scene/Cesium3DTilesetSpec.js]: Will there be a pull request into https://github.com/AnalyticalGraphicsInc/3d-tiles-tools for this new tileset?

Comment (ggetz) [reply Specs/Scene/Cesium3DTilesetSpec.js]: Created PR https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/149 for running changes. I would think review that last after the Cesium changes.

**Comment by ggetz** [comment]
Thanks @lilleyse , updated!

**Comment by lilleyse** [comment]
Thanks!

**Pull request merged**